win32ss/user/user32/misc/imm.c

```diff
@@ -33,43 +32,43 @@ Imm32ApiTable gImmApiEntries = {
 #include "immtable.h"
 };
 
-HRESULT WINAPI GetImmFileName(PWSTR lpBuffer, UINT uSize)
+HRESULT WINAPI User32GetImm32PathName(LPWSTR lpBuffer, size_t cchBuffer)
```

Comment:
This function seems to be an internal one only, so it doesn't need to be WINAPI.
```suggestion
static HRESULT
User32GetImm32PathName(
    _Out_ LPWSTR lpBuffer,
    _In_ size_t cchBuffer)
```

Reply:
Fixed.